<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMyTreeWidget</class>
 <widget class="QMainWindow" name="QMyTreeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2000</width>
    <height>1208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QMyTreeWidget</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="ScrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>1000</height>
     </size>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>798</width>
       <height>998</height>
      </rect>
     </property>
     <widget class="QLabel" name="Pictures">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>1000</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>1000</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>图片显示</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="TreeMenu">
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="title">
     <string>目录树</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <property name="toolTipsVisible">
     <bool>false</bool>
    </property>
    <addaction name="action_AddFolder"/>
    <addaction name="action_AddFile"/>
    <addaction name="action_DeleteItem"/>
    <addaction name="action_ScanItems"/>
    <addaction name="action_save"/>
    <addaction name="action_Close"/>
   </widget>
   <widget class="QMenu" name="ShowMenu">
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="action_ZoomIn"/>
    <addaction name="action_ZoomOut"/>
    <addaction name="action_ZoomRealSize"/>
    <addaction name="action_adjustheight"/>
    <addaction name="action_adjustwidth"/>
    <addaction name="action_DockVisible"/>
    <addaction name="action_DockFloat"/>
   </widget>
   <addaction name="TreeMenu"/>
   <addaction name="ShowMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_AddFolder"/>
   <addaction name="action_AddFile"/>
   <addaction name="action_DeleteItem"/>
   <addaction name="action_ScanItems"/>
   <addaction name="action_save"/>
   <addaction name="action_Close"/>
   <addaction name="separator"/>
   <addaction name="action_ZoomIn"/>
   <addaction name="action_ZoomOut"/>
   <addaction name="action_ZoomRealSize"/>
   <addaction name="action_adjustheight"/>
   <addaction name="action_adjustwidth"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>1100</width>
     <height>1100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1400</width>
     <height>1300</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="windowIcon">
    <iconset>
     <normalon>:/images/icons/open1.bmp</normalon>
    </iconset>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>目录</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QTreeWidget" name="treeWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1100</width>
       <height>1100</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1100</width>
       <height>1100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1400</width>
       <height>1300</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
      </font>
     </property>
     <property name="autoExpandDelay">
      <number>-1</number>
     </property>
     <property name="headerHidden">
      <bool>false</bool>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="headerVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="headerMinimumSectionSize">
      <number>650</number>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>650</number>
     </attribute>
     <attribute name="headerHighlightSections">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>图片文件夹</string>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignVCenter</set>
      </property>
      <property name="icon">
       <iconset resource="QMyTreeWidget.qrc">
        <normaloff>:/images/icons/Documents.ico</normaloff>:/images/icons/Documents.ico</iconset>
      </property>
     </column>
     <column>
      <property name="text">
       <string>文件类型</string>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignVCenter</set>
      </property>
      <property name="icon">
       <iconset resource="QMyTreeWidget.qrc">
        <normaloff>:/images/icons/Documents.ico</normaloff>:/images/icons/Documents.ico</iconset>
      </property>
     </column>
     <item>
      <property name="text">
       <string>旅游照片文件夹</string>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
      <property name="icon">
       <iconset resource="QMyTreeWidget.qrc">
        <normaloff>:/images/icons/folder1.bmp</normaloff>:/images/icons/folder1.bmp</iconset>
      </property>
      <property name="text">
       <string>旅游文档文件夹</string>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="icon">
       <iconset resource="QMyTreeWidget.qrc">
        <normaloff>:/images/icons/new1.bmp</normaloff>
        <normalon>:/images/icons/folder1.bmp</normalon>:/images/icons/new1.bmp</iconset>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
      <item>
       <property name="text">
        <string>北京</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
       <property name="checkState">
        <enum>Checked</enum>
       </property>
       <property name="icon">
        <iconset resource="QMyTreeWidget.qrc">
         <normaloff>:/images/icons/31.ico</normaloff>:/images/icons/31.ico</iconset>
       </property>
       <property name="text">
        <string>文档1</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/images/icons/new1.bmp</normalon>
        </iconset>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>上海</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="checkState">
        <enum>Checked</enum>
       </property>
       <property name="icon">
        <iconset resource="QMyTreeWidget.qrc">
         <normaloff>:/images/icons/31.ico</normaloff>:/images/icons/31.ico</iconset>
       </property>
       <property name="text">
        <string>文档2</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="icon">
        <iconset resource="QMyTreeWidget.qrc">
         <normaloff>:/images/icons/new1.bmp</normaloff>:/images/icons/new1.bmp</iconset>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>天津</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="checkState">
        <enum>Checked</enum>
       </property>
       <property name="icon">
        <iconset resource="QMyTreeWidget.qrc">
         <normaloff>:/images/icons/31.ico</normaloff>:/images/icons/31.ico</iconset>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
     </item>
     <item>
      <property name="text">
       <string>女朋友照片文件夹</string>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
      <property name="icon">
       <iconset resource="QMyTreeWidget.qrc">
        <normaloff>:/images/icons/folder1.bmp</normaloff>:/images/icons/folder1.bmp</iconset>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
      <item>
       <property name="text">
        <string>第一任女朋友</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="checkState">
        <enum>Checked</enum>
       </property>
       <property name="icon">
        <iconset resource="QMyTreeWidget.qrc">
         <normaloff>:/images/icons/31.ico</normaloff>:/images/icons/31.ico</iconset>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
        </font>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>第二任女朋友</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="checkState">
        <enum>Checked</enum>
       </property>
       <property name="icon">
        <iconset resource="QMyTreeWidget.qrc">
         <normaloff>:/images/icons/31.ico</normaloff>:/images/icons/31.ico</iconset>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
     </item>
    </widget>
   </widget>
  </widget>
  <action name="action_AddFolder">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="QMyTreeWidget.qrc">
     <normaloff>:/images/icons/folder1.bmp</normaloff>:/images/icons/folder1.bmp</iconset>
   </property>
   <property name="text">
    <string>添加目录</string>
   </property>
   <property name="toolTip">
    <string>添加目录</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="action_AddFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="QMyTreeWidget.qrc">
     <normaloff>:/images/icons/32.ico</normaloff>:/images/icons/32.ico</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建文件</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="action_DeleteItem">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="QMyTreeWidget.qrc">
     <normaloff>:/images/icons/delete1.bmp</normaloff>:/images/icons/delete1.bmp</iconset>
   </property>
   <property name="text">
    <string>删除节点</string>
   </property>
   <property name="toolTip">
    <string>删除节点</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="action_ScanItems">
   <property name="icon">
    <iconset resource="QMyTreeWidget.qrc">
     <normaloff>:/images/icons/fold.bmp</normaloff>:/images/icons/fold.bmp</iconset>
   </property>
   <property name="text">
    <string>遍历节点</string>
   </property>
   <property name="toolTip">
    <string>遍历节点</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="action_ZoomIn">
   <property name="icon">
    <iconset resource="QMyTreeWidget.qrc">
     <normaloff>:/images/icons/418.bmp</normaloff>:/images/icons/418.bmp</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="toolTip">
    <string>放大图片</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="action_ZoomOut">
   <property name="icon">
    <iconset resource="QMyTreeWidget.qrc">
     <normaloff>:/images/icons/416.bmp</normaloff>:/images/icons/416.bmp</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="toolTip">
    <string>缩小图片</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="action_ZoomRealSize">
   <property name="icon">
    <iconset resource="QMyTreeWidget.qrc">
     <normaloff>:/images/icons/414.bmp</normaloff>:/images/icons/414.bmp</iconset>
   </property>
   <property name="text">
    <string>恢复</string>
   </property>
   <property name="toolTip">
    <string>图片实际大小</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="action_adjustheight">
   <property name="icon">
    <iconset resource="QMyTreeWidget.qrc">
     <normaloff>:/images/icons/422.bmp</normaloff>:/images/icons/422.bmp</iconset>
   </property>
   <property name="text">
    <string>调整高度</string>
   </property>
   <property name="toolTip">
    <string>自动调整图片高度</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="action_adjustwidth">
   <property name="icon">
    <iconset resource="QMyTreeWidget.qrc">
     <normaloff>:/images/icons/424.bmp</normaloff>:/images/icons/424.bmp</iconset>
   </property>
   <property name="text">
    <string>调整宽度</string>
   </property>
   <property name="toolTip">
    <string>自动调整图片宽度</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="action_DockVisible">
   <property name="icon">
    <iconset resource="QMyTreeWidget.qrc">
     <normaloff>:/images/icons/preview1.bmp</normaloff>:/images/icons/preview1.bmp</iconset>
   </property>
   <property name="text">
    <string>窗体可见</string>
   </property>
   <property name="toolTip">
    <string>窗体可见</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+J</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="action_DockFloat">
   <property name="icon">
    <iconset resource="QMyTreeWidget.qrc">
     <normaloff>:/images/icons/814.bmp</normaloff>:/images/icons/814.bmp</iconset>
   </property>
   <property name="text">
    <string>窗体浮动</string>
   </property>
   <property name="toolTip">
    <string>添加目录</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+K</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="action_Close">
   <property name="icon">
    <iconset resource="QMyTreeWidget.qrc">
     <normaloff>:/images/icons/324.bmp</normaloff>:/images/icons/324.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Backspace</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset resource="QMyTreeWidget.qrc">
     <normaloff>:/images/icons/save1.bmp</normaloff>:/images/icons/save1.bmp</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存当前状态</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="QMyTreeWidget.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Close</sender>
   <signal>triggered()</signal>
   <receiver>QMyTreeWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
